Change pipeline to pull tiering flag from concourse

jk132 opened the pull request:
Change tiering logic flag in all concourse jobs to pull the value from the
corresponding concourse variable.
This allows the feature to be toggled in each environment by changing the
variable and restaging the app, rather than requiring a code change

whpearson commented:
Looks good, the mixed casing of the variable names _- jars a bit, but I see why you did it.

sanisc commented:
LGTM

jk132 commented:
As suggested, changed format of params to be more consistent with others (i.e. dash separated between all words).
Changed the params in Concourse to reflect this.
